src/Fhir.Anonymizer.Tool/FilesAnonymizerForJsonFormatResource.cs

```diff
@@ -18,21 +18,24 @@ public class FilesAnonymizerForJsonFormatResource
         private bool _validateInput;
         private bool _validateOutput;
         private string _configFilePath;
+        private bool _skipExistedFile;
 
         public FilesAnonymizerForJsonFormatResource(
             string configFilePath,
             string inputFolder,
             string outputFolder,
             bool isRecursive,
             bool validateInput,
-            bool validateOutput)
+            bool validateOutput,
+            bool skipExistedFile)
```

Comment:
Should we use some options object here to reduce the number of args?

Reply:
Refactor this file and add options.